### DISCUSSION TITLE
Error in the main window by every start

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#409] [General]
Hi,

I got an error in the main windw by every startup. The server still works, but I wonder what it is?

Kind regards,
Henk - ftp.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/ca0a0c47-2ad9-49a6-8555-e1e2e0fe82bf)

**Comment by rejetto** [comment]
hey! Can yo try to run hfs with *--dev* parameter? We may get extra useful information.

**Comment by PH5HP** [comment]
Hi,

Started HFS with --dev parameter. Two screenshot containing all output on startup of HFS.

Kind regards,
Henk - ftp.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/7f4f2d64-473c-4947-ae05-d00cc93fc779)
![image](https://github.com/rejetto/hfs/assets/140949730/691c5a71-3435-4934-9170-bd97d6b4b951)

**Comment by rejetto** [comment]
that has been possibly helpful, as the errors seems to be related to the loading of the certificate+key files.
Thanks, I'll let you know asap

**Comment by rejetto** [comment]
would you give a try to this version?
[hfs-windows.zip](https://github.com/rejetto/hfs/files/13573696/hfs-windows.zip)

if this doesn't solve, would you report your config using *Admin > options > edit config file > copy....*

**Reply by PH5HP** [reply]
Hi,

I see no errors. Here is the screenshot of the normal output.
![image](https://github.com/rejetto/hfs/assets/140949730/29c119cb-452f-402c-83cb-f1cc73cb9abf)

**Reply by rejetto** [reply]
sweeet, thanks a lot
